bin/admin/install-solvcon.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+if [ $# -lt 1 ]
+then
+    echo "Usage : $0 <solvcon env configuration file>"
+    exit
+fi
+solvcon_env_file=$1
+git clone https://github.com/solvcon/solvcon.git
+
+# you never know the env is ready to use SOLVCON or not, so
+# enable the env anyway.
+source $solvcon_env_file
+# basic env
+export SCSRC=$HOME/solvcon
+# prepare the ssh configuration to test rpc features on the localhost
```

Comment:
Sorry I had a typo "don'" in my comment.  No, I meant ssh key generation should be a standalone operation.  Now the architecture of the GCE scripts is still unclear, so I think it's probably good to separate functions apart.  A standalone feature like ssh key generation could use its own command, maybe like ``gce-ssh-keygen.sh``?  It will allow the scripts to serve three different scenarios:

1. Bare initialization of a single GCE node for maintenance.  For example, when we want to update the conda bucket, no remote SSH nor SOLVCON is needed.  The GCE scripts themselves are sufficient.
2. Initialization of GCE nodes for parallel SOLVCON execution.  ``gce-ssh-keygen.sh`` will need to be invoked along with the SOLVCON installation script.
3. Initialization of GCE nodes for other computing that needs SSH remote.